src/main/java/org/kpmp/slides/Stain.java

```diff
@@ -0,0 +1,45 @@
+package org.kpmp.slides;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "stains")
+public class Stain {
+
+	private String id;
```

Comment:
Do we need any annotation here to indicate the ID is the primary key?

Reply:
If we were going to write back to the db, I'd say yes.  Since we are just viewing, I don't think it is absolutely necessary.  I can add it if you want.

Reply:
I don't think it is either - just wanted to be sure it wouldn't give Spring a stomach ache.  Looks good.